pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -3961,3 +3919,37 @@ in
   in
   lib.genAttrs nodePackageNames nodePackage2VimPackage
 )
+// (
+  let
+    luarocksPackageNames = [
+      "fidget-nvim"
+      "gitsigns-nvim"
+      "image-nvim"
+      "lsp-progress-nvim"
+      "luasnip"
+      "lush-nvim"
+      "lz-n"
+      "lze"
+      "lzn-auto-require"
+      "middleclass"
+      "neorg"
+      "neotest"
+      "nui-nvim"
+      "nvim-cmp"
+      "nvim-nio"
+      "orgmode"
```

Comment:
We specifically left this as a neovim package because the Lua package didn't work.

Reply:
Hmm it doesn't work on unstable for me either, though with different error

Unsure what to do with this one

>  We need to test these instead of just assuming the functionality works because of differences between the sets in how they work

Assuming that latest commit works as well as stable release is also not correct

Reply:
> Hmm it doesn't work on unstable for me either, though with different error
> 
> Unsure what to do with this one

I dont use orgmode so I asked @mrcjkb to test the change. I forgot the change came from a PR about it possibly not working already on master... :S

Reply:
I tried adding orgmode to my config and it worked on this PR and on master